app/sidekiq/event_bus_gateway/letter_ready_email_job.rb

```diff
@@ -1,52 +1,94 @@
 # frozen_string_literal: true
 
 require 'sidekiq'
+require_relative 'constants'
 
 module EventBusGateway
   class LetterReadyEmailJob
     include Sidekiq::Job
-    include SentryLogging
 
-    sidekiq_options retry: 0
-    NOTIFY_SETTINGS = Settings.vanotify.services.benefits_management_tools
-    HOSTNAME_MAPPING = {
-      'dev-api.va.gov' => 'dev.va.gov',
-      'staging-api.va.gov' => 'staging.va.gov',
-      'api.va.gov' => 'www.va.gov'
-    }.freeze
+    include Constants
+
+    STATSD_METRIC_PREFIX = 'event_bus_gateway.letter_ready_email'
+
+    # retry for  2d 1h 47m 12s
+    # https://github.com/sidekiq/sidekiq/wiki/Error-Handling
+    sidekiq_options retry: 16
+
+    sidekiq_retries_exhausted do |msg, _ex|
+      job_id = msg['jid']
+      error_class = msg['error_class']
+      error_message = msg['error_message']
+      timestamp = Time.now.utc
+
+      ::Rails.logger.error('LetterReadyEmailJob retries exhausted',
+                           { job_id:, timestamp:, error_class:, error_message: })
+      tags = Constants::DD_TAGS + ["function: #{error_message}"]
+      StatsD.increment("#{STATSD_METRIC_PREFIX}.exhausted", tags:)
+    end
 
     def perform(participant_id, template_id)
-      notify_client.send_email(
-        recipient_identifier: { id_value: participant_id, id_type: 'PID' },
-        template_id:,
-        personalisation: { host: HOSTNAME_MAPPING[Settings.hostname] || Settings.hostname,
-                           first_name: get_first_name_from_participant_id(participant_id) }
-      )
+      if get_mpi_profile(participant_id)
+        response = notify_client.send_email(
+          recipient_identifier: { id_value: participant_id, id_type: 'PID' },
+          template_id:,
+          personalisation: { host: HOSTNAME_MAPPING[Settings.hostname] || Settings.hostname,
+                             first_name: get_first_name_from_participant_id(participant_id) }
+        )
+        EventBusGatewayNotification.create(user_account: user_account(participant_id), template_id:,
+                                           va_notify_id: response.id)
+        StatsD.increment("#{STATSD_METRIC_PREFIX}.success", tags: Constants::DD_TAGS)
+      end
     rescue => e
       record_email_send_failure(e)
     end
 
     private
 
     def notify_client
-      VaNotify::Service.new(NOTIFY_SETTINGS.api_key,
-                            { callback_klass: 'EventBusGateway::VANotifyEmailStatusCallback' })
+      @notify_client ||= VaNotify::Service.new(Constants::NOTIFY_SETTINGS.api_key,
```

Comment:
I think we can remove the Constants namespace here and it will also be consistent with HOSTNAME_MAPPING above.